README.md

```diff
@@ -51,6 +51,8 @@ So I decided to make a robust CLI with the proper error handling and testability
 
 As pytoil is a CLI program, I'd recommend installing with [pipx].
 
+`pipx install pytoil`
```

Comment:
If you do the code block as a multi-line thing GitHub will auto render a copy/paste button like this 👍🏻 : 
<img width="882" alt="image" src="https://user-images.githubusercontent.com/62767721/161828253-8f12dc9c-fa53-42f6-9d7f-9920b5cdb810.png">

Dont forget the closing '```' though it doesn't render on the suggestion


```suggestion
```shell
pipx install pytoil
```
```